src/spid-cookie.js

```diff
@@ -80,13 +80,24 @@ function clear(name) {
     clearVarnishCookie();
 }
 
+function getVarnishCookie() {
```

Comment:
Why not just use the get function instead? Or splitting that into a private cookie reading function and one handing the session stuff. This is code duplication.